lib/breathe_client.rb

```diff
@@ -65,6 +70,11 @@ def sickness_events(person:, after:)
           half_day_at_start = sickness[:half_start]
           half_day_at_end = sickness[:half_end]
 
+          if start_date < Date.parse(after)
+            puts "[DEBUG] #{person.label}: Skipping sickness with start date #{start_date.strftime("%F")} older than the requested earliest date #{after}"
```

Comment:
There's probably a better way to log in ruby, but it's probably not justified.

Reply:
If this runs for each absence someone has ever had, this could get very, very spammy, so maybe it is justified...

Reply:
I'm torn, because none of these should be showing up in the first place, if the Breathe API endpoint was responding the way it should. Currently it seems to be only absences that end up categorised as "other leave" that defy the `start_date` query, so more logging should hopefully produce more data to diagnose the issue. (Is it the API, is it the `breathe_ruby` gem, is it the rake task?)

Reply:
Yeah; I was mostly thinking "is there a nice library which would let us turn all the logging on/off really easily?", not "should we not have this logging"